presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -64,6 +70,7 @@
     private int domainCompactionThreshold = 100;
     private DataSize writerSortBufferSize = new DataSize(64, MEGABYTE);
     private boolean forceLocalScheduling;
+    private Set<NodeSelectionStrategy> supportedNodeSelectionStrategies = ImmutableSet.of();
```

Comment:
@highker: By reading the name of the config, I think this is trying to represent what is supported by Hive connector, in which case a set seems reasonable. I might be wrong though.

I'm actually a bit confused for this being a config. I think whether some node selection strategies are supported should be a property of a connector, rather than a configuration. Does this sound reasonable?